flow/controllers/base_controller.py

```diff
@@ -114,6 +114,8 @@ def get_action(self, env):
         # time step
         if accel is None:
             return None
+        else:
+            accel = min(max(accel, -self.max_deaccel), self.max_accel)
```

Comment:
this is already added into the Flow failsafes, so if you want it, use `fail_safe='feasible_accel'`

Reply:
thanks! removing